test/fixtures/rules/std_rule_cases/L018.yml

```diff
@@ -11,28 +11,19 @@ test_pass_with_clause_closing_oneline:
   # with statement oneline
   pass_str: with cte as (select 1) select * from cte
 
-test_fail_with_clause_closing_misaligned_indentation:
+test_pass_with_clause_closing_misaligned_indentation:
   # Fix with statement indentation
-  fail_str: |
+  pass_str: |
     with cte as (
         select 1
         ) select * from cte
 
-  fix_str: |
-    with cte as (
-        select 1
-    ) select * from cte
-
-test_fail_with_clause_closing_misaligned_negative_indentation:
+test_pass_with_clause_closing_misaligned_negative_indentation:
   # Fix with statement that has negative indentation
-  fail_str: |2
+  pass_str: |2
         with cte as (
         select 1
     ) select * from cte
```

Comment:
Again I see this as a regression due to this change - this should be fixed.

Reply:
Seems like it's okay because the `)` is on a line by itself, and that's all L018 will be responsible for after this PR.